src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -284,18 +263,10 @@ public virtual ContentResult Content(string content, string contentType, Encodin
             var result = new ContentResult
             {
                 Content = content,
+                ContentEncoding = contentEncoding,
+                ContentType = contentType
             };
 
-            if (contentType != null)
-            {
-                result.ContentType = contentType;
-            }
-
-            if (contentEncoding != null)
-            {
-                result.ContentEncoding = contentEncoding;
-            }
-
```

Comment:
just simplification, ContentResult just have auto properties for these values so null checking is meaningless